nucliadb/nucliadb/ingest/orm/processor.py

```diff
@@ -305,7 +300,7 @@ async def process(
             )
         elif self.audit is None:
             logger.warning("No audit defined")
-        elif audit_type is None:
+        elif audit_type is None and txn_result is not None:
```

Comment:
what about the case when both are None?

Reply:
Not possible, because audit_type will be none if txn_result is none, as is derived from it